lib/elixir/lib/hash_dict.ex

```diff
@@ -221,16 +221,35 @@ defmodule HashDict do
 end
 
 defimpl Enumerable, for: HashDict do
-  def reduce(dict, acc, fun),  do: HashDict.reduce(dict, acc, fun)
-  def member?(dict, {k, v}), do: {:ok, match?({:ok, ^v}, HashDict.fetch(dict, k))}
-  def member?(_dict, _),       do: {:ok, false}
-  def count(dict),             do: {:ok, HashDict.size(dict)}
+  def reduce(dict, acc, fun) do
+    # Avoid warnings about HashDict being deprecated.
+    hash_dict = HashDict
```

Comment:
I'm just gonna call it `module` now that you point it out, it's used super close and we don't need the excess verbosity.